images/build/Dockerfiles/full.Dockerfile

```diff
@@ -21,7 +21,8 @@ RUN ./build.sh golang     /opt/startupcmdgen/golang
 FROM buildpack-deps:${DEBIAN_FLAVOR}-curl
 ARG DEBIAN_FLAVOR
 ARG SDK_STORAGE_BASE_URL_VALUE="https://oryx-cdn.microsoft.io"
-ARG AI_KEY
+ARG AI_CONNECTION_STRING
+ENV ORYX_AI_CONNECTION_STRING =${AI_CONNECTION_STRING}
```

Comment:
Would you mind deleting the space between `ORYX_AI_CONNECTION_STRING` and `=`?

Reply:
Updated.